specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-10-15-preview/examples/CosmosDBSqlClientEncryptionKeyCreateUpdate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "api-version": "2021-10-15-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rgName",
+    "accountName": "accountName",
+    "databaseName": "databaseName",
+    "clientEncryptionKeyName": "cekName",
+    "createUpdateClientEncryptionKeyParameters": {
+      "properties": {
+        "resource": {
+          "encryptionAlgorithm": "AEAD_AES_256_CBC_HMAC_SHA256",
+          "wrappedDataEncryptionKey": "This is actually an array of bytes. This request/response is being presented as a string for readability in the example",
```

Comment:
>This is actually an array of bytes. This request/response is being presented as a string for readability in the example [](http://example.com/codeflow?start=39&length=119)

this is going to be a base64 encoded string, right?

Reply:
This will be byte[]